### DISCUSSION TITLE
blog/aws-s3-nestjs

### DISCUSSION THREAD

**Discussion by giscus** [adarshaacharya/portfolio-giscus#5] [Announcements]
# blog/aws-s3-nestjs

Store private and public files in single S3 bucket, generate pre-signed URLs for private files using NestJS and AWS SDK v3. Also, perform CRUD operations on S3 objects using NestJS.

https://www.adarsha.dev/blog/aws-s3-nestjs

**Comment by rabinabaga** [comment]
thank you so much

**Reply by adarshaacharya** [reply]
welcome
